tests/from.bats

```diff
@@ -255,7 +255,7 @@ load helpers
   _prefetch alpine
   run_buildah from --quiet --add-host=localhost:127.0.0.1 --pull --signature-policy ${TESTSDIR}/policy.json alpine
   cid=$output
-  run_buildah run $cid -- cat /etc/hosts
```

Comment:
@nalind, does this look hip to you?  This is my one worry spot with this change.  Debian and Alpine act differently.  If you take the "stock" alpine image and run with current upstream Buildah, the run command without the `--net-option` would retain the following in the containers /etc/host file in the containers storage on the host:
```
127.0.0.1	localhost localhost.localdomain
::1		localhost localhost.localdomain
```

And if you investigate the container when it's running, you first see the "# Buildah Generated" lines, then these two lines.  However, the Buildah Generated lines are not retained on disk for the container, only these two lines.

Debian however, has nothing except the "# Buildah Generated" lines when it too is running and then nothing in the /etc/host in the container storage on the host.

With these changes, that file in container storage is empty for both distros.  In both this variant of Buildah and current upstream, the host specified in --add-host is not retained when the run command completes.

Reply:
Do a buildah mount and see what the contents of the /etc/hosts file are. these hosts files saved to the image, are from the underlying image, which is why you are seeing the difference I believe.  
I don't know why in a running container the localhost data provided by Buildah would be different.  
I don't think we ever save the contents of /etc/hosts or /etc/resolv.conf into a container image.

Reply:
If I do `buildah run {alpine||debian}-working-container cat /etc/hosts`  on both distros I see:

```
# Generated by Buildah
127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4
::1         localhost localhost.localdomain localhost6 localhost6.localdomain6
127.0.0.1   0f8200b7e866
::1         0f8200b7e866
```
If I mount them, then two different but similar lines in the alpine mount, and none in Debian.

```
# ma=$(buildah mount alpine-working-container)
# mb=$(buildah mount debian-working-container)

# cat $ma/etc/hosts
127.0.0.1	localhost localhost.localdomain
::1		localhost localhost.localdomain

# cat $mb/etc/hosts

```
So it looks like we're getting something from alpine directly and it's not part of the generated one that we make when running.  I'm not quite sure how/where that's coming from.

The main focus of this fix is just to retain whatever the user adds to /etc/hosts if `--net=none`